tls/README.md

```diff
@@ -1,15 +1,30 @@
 # Temporal with tls enabled dependencies
 
-## Setup
+## Execute
 
 run from a shell
 
-`./tls/make-certs.sh`
+`./tls/run-tls.sh`
 
-## Startup
+## Script source with comments
```

Comment:
Not sure about copying the script in here since we'll inevitably forget to keep it in sync after changes. I guess it's more of living docs than a script though

Reply:
I need markdown transclusion

Reply:
put the script in between sentinel comments in README.md and then have `run-tls.sh` do
```
sed -ne '/START/,/END/p' < README.md | sh
```
:stuck_out_tongue_closed_eyes: